react/hooks/useShippingOptions.ts

```diff
@@ -138,9 +139,7 @@ const useShippingOptions = () => {
 
     setSelectedZipCode(inputZipCode)
 
-    await updateSession(inputZipCode, coordinates)
-
-    await fetchPickups(countryCode, inputZipCode, coordinates)
+    await updateSession(countryCode, inputZipCode, coordinates)
```

Comment:
Can this be updateSession be done in a single call? 

If not, perhaps you can make those calls in parallel use a single await Promise.all